Ontology/Capability/Sensor/StatusSensor/FanStatusSensor.json

```diff
@@ -1,6 +1,15 @@
 {
   "@id": "dtmi:digitaltwins:rec_3_3:core:FanStatusSensor;1",
   "@type": "Interface",
-  "extends": "dtmi:digitaltwins:rec_3_3:core:StatusSensor;1",
+  "displayName": {
+    "en": "Fan Status Sensor"
+  },
+  "extends" : [
+    "dtmi:digitaltwins:rec_3_3:core:StatusSensor;1",
+    "dtmi:digitaltwins:rec_3_3:core:FanStatusSensor;1"
```

Comment:
FanStatusSensor should not extend itself. Same comment applies to other interfaces in this PR.

Reply:
Fixed. I will review them all. It's a copy-paste mistake. Thanks for catching that.